comp/dogstatsd/server/server.go

```diff
@@ -144,6 +146,29 @@ type server struct {
 	enrichConfig enrichConfig
 }
 
+func init() {
+	expvarcollector.RegisterExpvarReport("dogstatsdStats", func() (interface{}, error) {
+		dogstatsdStats := make(map[string]interface{})
+
+		dogstatsdStatsJSON := []byte(dogstatsdExpvars.String())
+		dogstatsdUdsStatsJSON := []byte(expvar.Get("dogstatsd-uds").String())
+		dogstatsdUDPStatsJSON := []byte(expvar.Get("dogstatsd-udp").String())
+		json.Unmarshal(dogstatsdStatsJSON, &dogstatsdStats) //nolint:errcheck
+		dogstatsdUdsStats := make(map[string]interface{})
+		json.Unmarshal(dogstatsdUdsStatsJSON, &dogstatsdUdsStats) //nolint:errcheck
+		for name, value := range dogstatsdUdsStats {
+			dogstatsdStats["Uds"+name] = value
+		}
+		dogstatsdUDPStats := make(map[string]interface{})
+		json.Unmarshal(dogstatsdUDPStatsJSON, &dogstatsdUDPStats) //nolint:errcheck
```

Comment:
This applies to the rest of the changes but it's a bit iffy that we do `expvar -> string -> bytes -> json -> interface{}` conversions. I think we may need strongly typed return values or avoid doing the conversions in the callbacks. Maybe some of [this](https://github.com/DataDog/datadog-agent/blob/main/pkg/collector/runner/expvars/expvars.go) code can be used for reference.